buildspec.yml

```diff
@@ -39,16 +44,22 @@ phases:
       - CHECKED_IN_VERSION=`cat /go/src/github.com/aws/amazon-ecs-cli/VERSION`
       - echo "VERSION_FILE=$CHECKED_IN_VERSION"
       - echo "GOPATH=$GOPATH"
+      # Check if there is a tag currently pointing at the head and exit if not. 
+      - |
+        if [ -z "$VERSION" ]; then
+          echo "the current commit is not tagged. Stopping the build stage..."
+          exit 1
+        fi
```

Comment:
Did we miss this previously? This seems like a generic validation improvement

Reply:
Yes, it seems like we thought that the check against the checked in version would cover this case, but the pipeline stage was succeeding when there was no git tag. I thought this would be a good time to add that validation to make sure it behaves the way we thought it did.